.travis/install.sh

```diff
@@ -75,6 +75,10 @@ else
             tar zxf openssl-$OPENSSL_VERSION_NUMBER.tar.gz
             cd openssl-$OPENSSL_VERSION_NUMBER
             ./config shared no-asm no-ssl2 -fPIC --prefix="$HOME/$OPENSSL_DIR"
+            # modify the shlib version to a unique one to make sure it doesn't load the system one.
+            sed -i "s/^SHLIB_MAJOR=[0-9]/SHLIB_MAJOR=100/" Makefile
+            sed -i "s/^SHLIB_MINOR=[0-9].[0-9]/SHLIB_MINOR=0.0/" Makefile
```

Comment:
This code should probably be a bit more resillient, on some releases `SHLIB_MINOR` doesn't have a period and `SHLIB_VERSION_NUMBER` only has a single period. Also teh `.` should be escaped.